client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -409,7 +409,65 @@ const Editor = (props: EditorProps): JSX.Element => {
         });
       }
     });
+    // Introduced as a work around for a bug in JAWS 2022
+    // https://github.com/FreedomScientific/VFO-standards-support/issues/598
+    editor.addAction({
+      id: 'toggle-aria-roledescription',
+      label: 'Toggle aria-roledescription',
+      keybindings: [
+        monaco.KeyMod.CtrlCmd | monaco.KeyMod.Alt | monaco.KeyCode.KEY_J
+      ],
+      run: toggleAriaRoledescription
+    });
     editor.onDidFocusEditorWidget(() => props.setEditorFocusability(true));
+
+    // aria-roledescription is on (true) by default, check if it needs
+    // to be removed.
+    const ariaRoledescription = getStoredAriaRoledescription();
+    if (!ariaRoledescription) {
+      setAriaRoledescription(false);
+    }
+  };
+
+  const toggleAriaRoledescription = () => {
+    const newRoledescription = !getStoredAriaRoledescription();
+    setAriaRoledescription(newRoledescription);
+    ariaAlert(
+      `aria-roledescription has been turned ${
+        newRoledescription ? 'on' : 'off'
+      }`
+    );
+  };
+
+  const setAriaRoledescription = (value: boolean) => {
+    const textareas = document.querySelectorAll('.monaco-editor textarea');
+    textareas.forEach(textarea => {
+      value
+        ? textarea.setAttribute('aria-roledescription', 'editor')
+        : textarea.removeAttribute('aria-roledescription');
+    });
+    store.set('ariaRoledescription', value);
+  };
+
+  const getStoredAriaRoledescription = (): boolean => {
+    const roledescription = store.get('ariaRoledescription') as
+      | boolean
+      | undefined;
+    return !!(roledescription === undefined || roledescription);
+  };
```

Comment:
```suggestion
  const getStoredAriaRoledescription = () => !!store.get('ariaRoledescription');
```
With something this small it's pretty clear that it returns a boolean.  Also, we don't need to worry about the type casts or undefined checking, since !! takes care of all of that.

Reply:
Yes, it should default to true when the ariaRoledescription doesn't exist in local storage. I guess it's so new that I just didn't think about using nullish coalescing but it's a perfect fit here.